.vscodeignore

```diff
@@ -15,7 +15,11 @@ coverage/
 **/*.map
 **/*.ts
 **/*.gif
```

Comment:
why do we need to ignore gif?

Reply:
this file indicated the file which should be ignored when packaging the extension, anything which is not required in the runtime should be ignored to make the install package as small as possible.

Reply:
Can we use sth. like:


# Ignore everything
*

# Whitelist what you need
!sr

Not sure whether that's possible, but just an idea.

---
In reply to: [373910436](https://github.com/docascode/vscode-docs-build/pull/32#discussion_r373910436) [](ancestors = 373910436)

Reply:
does that nessecery? this is the same design as `.gitignore` file.

Reply:
That works, good idea